app/_data/docs_nav_gateway_3.4.x.yml

```diff
@@ -29,8 +29,10 @@ items:
         absolute_url: true
       - text: Breaking Changes	
         items:
-          - text: Kong Gateway 3.4.x	
+          - text: List of versions
```

Comment:
Is this page meant to be as a "redirect"/placeholder since the old pages used to point here? Since we already list the breaking changes by version below, I don't think we need this in the nav. Could we keep the page so people who follow that old link end up here but the page is unlisted in the nav?

Reply:
From Lena:

> how about turning the breaking changes section into a "book" so that it gets next/previous links? And then maybe use them in variables at the top of each breaking changes page as well.

I like this idea, I think it would work!